lib/child_process.js

```diff
@@ -318,6 +323,8 @@ exports.execFile = function(file /*, args, options, callback*/) {
       stdoutLen += encoding ? Buffer.byteLength(chunk, encoding) : chunk.length;
 
       if (stdoutLen > options.maxBuffer) {
+        // TODO helpful to convert to ERR_BUFFER_OUT_OF_BOUNDS?
+        // can't specify stderr/stdout with that new error.
         ex = new Error('stdout maxBuffer exceeded');
```

Comment:
Would have to create `ERR_CHILD_PROCESS_STDERR_MAXBUFFER` in addition to `ERR_CHILD_PROCESS_STDOUT_MAXBUFFER`, is this ok?